godotproject/enemy-Ant.gd

```diff
@@ -57,9 +63,10 @@ func launch_attack():
 		
 		$ActionDelay.start()
 
-func _on_PlayerDetector_body_entered(body):
-	if attackState == STATE.idle_e:
-		begin_attack()
+func _on_PlayerDetector_body_entered(body):	
+	if body.name == "Player":		
```

Comment:
Should we have a better way of checking if an object is the player? Is there any reason why we'd want to make it more complicated than this? @fheysel

Reply:
We could also give the player an attribute or function called "takes_aggro" or something that represents the actual behaviour, instead of just checking if it's the player. This would lead to a lot of stuff needing to be declared on the player object, but then it'd be super easy to make other objects targetable by enemies

Reply:
I guess if we need that in the future, we can refactor then

Reply:
Yeah I think it might be a good idea to make it more readable in the future, for now lets just focus on release